INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:22:22] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11, metrics endpoint was reached
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:22:58] "GET /metrics HTTP/1.1" 200 -
DEBUG:root:2021-08-11 18:25:36.756925, hello endpoint was reached
DEBUG:root:2021-08-11 18:26:14.729762, hello endpoint was reached
DEBUG:root:2021-08-11 18:26:14.730761, status endpoint was reached
DEBUG:root:2021-08-11 18:26:14.731761, metrics endpoint was reached
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:26:17] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:26:47] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:26:47] "GET /favicon.ico HTTP/1.1" 404 -
DEBUG:root:2021-08-11 18:27:29.685136, status endpoint was reached
DEBUG:root:2021-08-11 18:27:29.685136, metrics endpoint was reached
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11 18:28:23.975996, metrics endpoint was reached
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2097, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:28:23] "GET /metrics HTTP/1.1" 500 -
DEBUG:root:2021-08-11 18:28:28.677333, metrics endpoint was reached
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2097, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:28:28] "GET /metrics HTTP/1.1" 500 -
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:28:29] "GET /favicon.ico HTTP/1.1" 404 -
DEBUG:root:2021-08-11 18:28:30.578460, metrics endpoint was reached
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2097, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:28:30] "GET /metrics HTTP/1.1" 500 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11 18:28:49.334449, test endpoint was reached
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2097, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:28:49] "GET /metrics HTTP/1.1" 500 -
DEBUG:root:2021-08-11 18:28:50.555130, test endpoint was reached
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2097, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 18:28:50] "GET /metrics HTTP/1.1" 500 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11 19:20:53.912908, test endpoint was reached
ERROR:app:Exception on /metrics [GET]
Traceback (most recent call last):
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\python\envs\envCurso\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\python\envs\envCurso\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app.py", line 15, in log
    return f(args, kwargs)
TypeError: metrics() takes 0 positional arguments but 2 were given
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:20:53] "GET /metrics HTTP/1.1" 500 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11 19:22:25.361024, test endpoint was reached
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:22:25] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11 19:23:41.250010, metrics endpoint was reached
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:23:41] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:23:54] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:24:18] "GET /status HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
DEBUG:root:2021-08-11 19:28:10.826720, status endpoint was reached
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:28:10] "GET /status HTTP/1.1" 200 -
DEBUG:root:2021-08-11 19:28:22.623135, metrics endpoint was reached
INFO:werkzeug:127.0.0.1 - - [11/Aug/2021 19:28:22] "GET /metrics HTTP/1.1" 200 -
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
2021-08-11 19:30:49.702737, metrics endpoint was reached
127.0.0.1 - - [11/Aug/2021 19:30:49] "GET /metrics HTTP/1.1" 200 -
